Tests/SwiftSMTPTests/TestHeaderFolding.swift

```diff
@@ -0,0 +1,138 @@
+/**
+ * Copyright Stephen Beitzel 2021
+ *
```

Comment:
On a non-technical note, I need to look into what the norm here for open source projects regarding copyrights of contributions.  I thought it was typical for copyright to belong to the project.  But Kitura's a bit of a different case since a lot of the existing files are copyright by IBM, so I understand following that pattern.  Do you have any experience or strong feelings about this?

Reply:
It should be whatever the project community agrees on. For obvious legacy reasons, most of the code has a copyright header for IBM, but I understand it's now been spun out from IBM to exist on its own. As part of that process, did anyone set up an entity for "The Kitura Project"?

My own feeling is, I'm happy to assign copyright to Kitura, or to hold it myself but license it under the Apache license since that's the project's license, but it feels weird to assign it to IBM since they've said they're done sponsoring the project.

Reply:
I've been updating existing IBM files to say "Copyright: IBM and Kitura project authors".  So if you want to assign it to Kitura or The Kitura Project, AFAIK this is sufficient.  I'll look around and see what other open source projects are doing.  I was hoping to avoid starting a non-profit, but if we need a more definitive legal existence, I can look into that.  Either way I agree it wouldn't make sense to assign it to IBM at this point.